media/videostitcher/create_cdn_key_akamai.go

```diff
@@ -14,52 +14,52 @@
 
 package videostitcher
```

Comment:
Same as previous.

Reply:
I needed to comment out some entire files because they now break due to the updated library version. I'll uncomment and update them in a separate PR.